yb-voyager/src/tgtdb/oracle.go

```diff
@@ -570,9 +570,9 @@ func (tdb *TargetOracleDB) importBatch(conn *sql.Conn, batch Batch, args *Import
 	var errbuf string
 	outbuf, errbuf, err = sqlldr.RunSqlldr(sqlldrArgs, password)
 
-	if outbuf == "" && errbuf == "" && err != nil {
+	if errbuf != "" || err != nil {
```

Comment:
I would guess that only `err != nil` should be required. Are you sure that there are failure cases where `err == nil` and `errbuf != ""`? Does `errbuf != ""` really mean there was any error while executing sqlldr?

Reply:
yes you are right only `err != nil` is required but I guess we should print the `errbuf` in the err msg.